Projects/Scripts/Misc/Email.cs

```diff
@@ -1,98 +1,154 @@
+using MailKit.Net.Smtp;
+using MimeKit;
+using Server.Accounting;
+using Server.Engines.Help;
 using System;
-using System.Net;
-using System.Net.Mail;
+using System.IO;
 using System.Text.RegularExpressions;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace Server.Misc
 {
   public class Email
   {
-    /* In order to support emailing, fill in EmailServer and FromAddress:
-     * Example:
-     *  public static readonly string EmailServer = "mail.domain.com";
-     *  public static readonly string FromAddress = "runuo@domain.com";
-     * 
-     * If you want to add crash reporting emailing, fill in CrashAddresses:
-     * Example:
-     *  public static readonly string CrashAddresses = "first@email.here,second@email.here,third@email.here";
-     * 
-     * If you want to add speech log page emailing, fill in SpeechLogPageAddresses:
-     * Example:
-     *  public static readonly string SpeechLogPageAddresses = "first@email.here,second@email.here,third@email.here";
-     */
-
-    public static readonly string EmailServer = null;
-    public static readonly int EmailPort = 25;
-
-    public static readonly string FromAddress = null;
-    public static readonly string EmailUsername = null;
-    public static readonly string EmailPassword = null;
+    public static readonly MailboxAddress FromAddress = null; //new MailboxAddress("Joey Tribbiani", "joey@friends.com");

```

Comment:
Let's move these settings to the modernuo.json file?

We may need to update the modernuo.json file to be a more idiomatic configuration system too.